src/index.js

```diff
@@ -52,9 +44,10 @@ export default function createStore(state) {
 
 			// Note: perf tests verifying this implementation: https://esbench.com/bench/5a295e6299634800a0349500
 			return function() {
-				let args = [state];
-				for (let i=0; i<arguments.length; i++) args.push(arguments[i]);
-				let ret = action.apply(this, args);
+				const args = [state];
+				const l = arguments.length;
```

Comment:
Hoisting this adds bytes and can actually hurt performance.

mraleph has a great blog post describing the effect in detail here:
http://mrale.ph/blog/2014/12/24/array-length-caching.html

Reply:
I'll agree with these points.